pythran/analyses/aliases.py

```diff
@@ -245,6 +253,11 @@ def visit_FunctionDef(self, node):
                             for arg in node.args.args)
 
         self.generic_visit(node)
+        if '@' in self.aliases:
+            param_indices = {i for i in range(len(node.args.args))
+                             if node.args.args[i].id in self.aliases}
+            node.return_alias = lambda node: {node.args[i]
+                                              for i in param_indices}
```

Comment:
Just to be sure: You compute param_indices based on "node" but here, you use these indices based on another "node" (the lambda one, not the function one). Is it normal?